.env.sample

```diff
@@ -0,0 +1,32 @@
+# Set a unique secret key for the project, required for running outside DEBUG mode
+SECRET_KEY=Ensure_you_set_a_secret_key_this_is_important!
+
+LOCAL_WEB_PORT=5000
+
+# Database details for postgres container
+DB_TYPE=postgresql
+DB_NAME=db
+DB_USER=root
+DB_PASSWD=<need_to_set>
+DB_HOST=127.0.0.1
+DB_PORT=5432
+
+REDIS_URI=redis://redis:6379
+ESEARCH_URI=esearch
+
+UPLOAD_FOLDER=<need_to_set>
+PURCHASE_URI=<need_to_set>
+
+BABEL_DEFAULT_LOCALE=en_US
+BABEL_DEFAULT_TIMEZONE=UTC
+BABEL_TRANSLATION_DIRECTORIES=../translations
+BABEL_CURRENCY=USD
+
+MAIL_SERVER=<need_to_set>
+MAIL_PORT=<need_to_set>
+MAIL_TLS=<need_to_set>
+MAIL_USERNAME=<need_to_set>
+MAIL_PASSWORD=<need_to_set>
+MAIL_DEFAULT_SENDER=<need_to_set>
+GA_MEASUREMENT_ID=<need_to_set>
+
```

Comment:
pls set predefined values as samples

Reply:
I will update this, when I updated account activation - in the next request